frontend/src/metabase/public/components/EmbedFrame/EmbedFrame.tsx

```diff
@@ -37,6 +37,7 @@ interface OwnProps {
   description?: string;
   dashboard?: Dashboard;
   actionButtons?: JSX.Element[];
+  footerVariant?: "default" | "big";
```

Comment:
Saw `Variant` is being exported in `frontend/src/metabase/public/components/EmbedFrame/LogoBadge.styled.tsx`. Should we reuse that?